src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/overview/overview.component.scss

```
@@ -0,0 +1,6 @@
+::ng-deep cd-table-actions.float-right ul {
+  top: 100% !important;
+  right: 0 !important;
+  bottom: auto !important;
+  left: auto !important;
```

We should limit this to `cd-mirroring`, to prevent affecting other components:
```suggestion
::ng-deep cd-mirroring cd-table-actions.float-right ul {
  right: 0 !important;
  left: auto !important;
```
`top `and `bottom `are not changed, so we can remove them.